C-Plus-Plus/stack/next_greatest_right.cpp

```diff
@@ -0,0 +1,62 @@
+/*Next greatest element to the right of each element in the array
+  ALGORITHM
+ -->Using a stack to find out the next greatest element to the right
+ -->Start the scanning of the array from the right 
+ -->Declare an array and a stack
+ -->If the stack is empty, then no element to the right is greater than the present element, for example, for the last element
+ -->If the stack is not empty but the element present at the top of the stack is not greater than the current element then, pop the stack till either the stack is not empty or the top element of stack is greater than the current element
+ -->If the stack is not empty and the element at the top of the stack is greater than the current element then push the topmost element to the stack
+*/
```

Comment:
Don't use ```-->``` unnecessarily. Instead, use Bullet Points. Also, keep the lines shorter so that they can be read and understood easily

Reply:
like having 79 charachters in a line is a norm

Reply:
@gat786 Yes, I have made the changes.
@HarshCasper @gat786 Please review it